### PR TITLE
fix: wire up cron again

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

File: cmd/ftl-cron/main.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		kong.UsageOnError(),
+		kong.Vars{"version": ftl.Version, "timestamp": time.Unix(t, 0).Format(time.RFC3339)},
+	)
+	cli.ProvisionerConfig.SetDefaults()
+
+	ctx := log.ContextWithLogger(context.Background(), log.Configure(os.Stderr, cli.LogConfig))
+	err = observability.Init(ctx, false, "", "ftl-cron", ftl.Version, cli.ObservabilityConfig)
+	kctx.FatalIfErrorf(err, "failed to initialize observability")
+
+	verbClient := rpc.Dial(ftlv1connect.NewVerbServiceClient, cli.ProvisionerConfig.ControllerEndpoint.String(), log.Error)
```
Comment (alecthomas) [inline cmd/ftl-cron/main.go:46]: Verb calls shouldn't be routing through the Provisioner...?

File: cmd/ftl-cron/main.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	"github.com/TBD54566975/ftl/internal/log"
+	"github.com/TBD54566975/ftl/internal/observability"
+	"github.com/TBD54566975/ftl/internal/rpc"
+)
+
+var cli struct {
+	Version             kong.VersionFlag     `help:"Show version."`
+	ObservabilityConfig observability.Config `embed:"" prefix:"o11y-"`
+	LogConfig           log.Config           `embed:"" prefix:"log-"`
+	ProvisionerConfig   provisioner.Config   `embed:""`
```
Comment (alecthomas) [inline cmd/ftl-cron/main.go:26]: Verb calls should be routing through the Controller.

**Review by alecthomas — APPROVED** [review]
LGTM except for using the provisioner for routing calls.

File: cmd/ftl-cron/main.go (hunk 1 above)
Comment (stuartwdouglas) [reply cmd/ftl-cron/main.go]: oops, that was a copy/paste error I should have picked up.

File: cmd/ftl-cron/main.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	"github.com/TBD54566975/ftl/internal/log"
+	"github.com/TBD54566975/ftl/internal/observability"
+	"github.com/TBD54566975/ftl/internal/rpc"
+)
+
+var cli struct {
+	Version             kong.VersionFlag     `help:"Show version."`
+	ObservabilityConfig observability.Config `embed:"" prefix:"o11y-"`
+	LogConfig           log.Config           `embed:"" prefix:"log-"`
+	CronConfig          cron.Config          `embed:""`
```
Comment (alecthomas) [inline cmd/ftl-cron/main.go:25]: In general we should prefix all embedded configs.

Comment (stuartwdouglas) [reply cmd/ftl-cron/main.go:25]: That is not the pattern we have been using for the 'main' config though. Cron, Runner, Provisioner, Controller etc all have one 'main' config that is not prefixed.

**Pull request merged**